README.md

```diff
@@ -20,9 +20,13 @@ In a nutshell, Protocol Buffers have two main functions:
 - They are a language for writing schemas for your data.
 - They define a binary format for serializing your data.
 
-These two independent traits functions work together to allow your project and everyone who interacts with it to define messages, fields, and service APIs in the exact same way. In a practical sense as it relates to **Protobuf-ES**, this means no more disparate JSON types all over the place. Instead, you define a common schema in a Protobuf file, such as:
+These two independent traits functions work together to allow your project and everyone who interacts with it to define
```

Comment:
This should probably be `...independent traits function together` or `...work together`. Looks like there was always an extra word in there.

Reply:
Yep, https://github.com/bufbuild/protobuf-es/pull/907/commits/b0dbff296094dfe68e6f5434bce36c198a7620a5.